enterprise/server/scheduling/scheduler_server/scheduler_server.go

```diff
@@ -147,6 +149,29 @@ func (en *executionNode) GetExecutorID() string {
 	return en.executorID
 }
 
+// TODO: Expand interfaces.ExecutionNode interface to include all executionNode
```

Comment:
nit: please assign this TODO to a person or bug. A la:

TODO(bduffs): do the thing OR
TODO(b/1w123123): do the thing